zerver/webhooks/wordpress/doc.md

```diff
@@ -1,33 +1,52 @@
+# Zulip WordPress integration
+
 Get WordPress notifications in Zulip!
 
-If you're hosting your WordPress blog yourself (i.e., not on WordPress.com),
-first install the
-[HookPress plugin](https://wordpress.org/plugins/hookpress/) (experimental).
+!!! tip ""
```

Comment:
The tips in some other documents are included at the top of the tabbed section, but folks felt this doc looks better this way.

Reply:
This plugin has been closed since July 2019 for a security reason, so makes more sense to remove this entirely.